webpack.config.production.js

```diff
@@ -1,23 +1,23 @@
 /* eslint-disable */
 
-var path = require("path");
-var webpack = require("webpack");
+var path = require('path');
```

Comment:
GLOBAL: `eslint-config-defaults` uses double quotes. We should revert _and_ get lint turned on for all JS files in this repo. (Can be separate file).